File: algo-roman-numerals/python/roman_numerals_spec.py
from roman_numerals import to_roman

# lazy 
# print(to_roman(1) == 'I')
# print(to_roman(3) == 'III')
# print(to_roman(4) == 'IIII')
# print(to_roman(1254) == 'MCCLIIII')
# print(to_roman(2890) == 'MMDCCCLXXXX')
# add tests to cover different edge cases

# modern
print(to_roman(1) == 'I')
print(to_roman(3) == 'III')
print(to_roman(100) == 'C')
print(to_roman(1254) == 'MCCLIV')
print(to_roman(2890) == 'MMDCCCXC')

File: algo-roman-numerals/js/romanNumeralsSpec.js
var rn = require("./romanNumerals");

console.log(rn.toRoman(1) === 'I');
console.log(rn.toRoman(3) === 'III');
console.log(rn.toRoman(4) === 'IV');
console.log(rn.toRoman(9) === 'IX');
console.log(rn.toRoman(14) === 'XIV');
console.log(rn.toRoman(44) === 'XLIV');
console.log(rn.toRoman(150) === 'CL');
console.log(rn.toRoman(944) === 'CMXLIV');

for (let i = 1; i <= 3999; i++) {
    console.log(i);
    console.log(rn.toRoman(i));
}